commit 57659a9ad08107419b2346c941d2d306f5e4f23f (refs/changes/17/3117/3)
Author: Robert Mustacchi <rm@joyent.com>
Date:   2018-01-05T19:10:10+00:00 (1 year, 9 months ago)
    
    OS-6528 Need support for mpt_sas tri-mode parts
    Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com>
    Approved by: Jerry Jelinek <jerry.jelinek@joyent.com>

diff --git a/overlay/generic/etc/driver_aliases b/overlay/generic/etc/driver_aliases
index b0b10436..2ebfc9cb 100644
--- a/overlay/generic/etc/driver_aliases
+++ b/overlay/generic/etc/driver_aliases
@@ -927,8 +927,25 @@ mpt_sas "pciex1000,94"
 mpt_sas "pciex1000,95"
 mpt_sas "pciex1000,96"
 mpt_sas "pciex1000,97"
+mpt_sas "pciex1000,aa"
+mpt_sas "pciex1000,ab"
+mpt_sas "pciex1000,ac"
+mpt_sas "pciex1000,ad"
+mpt_sas "pciex1000,ae"
+mpt_sas "pciex1000,af"
+mpt_sas "pciex1000,c0"
+mpt_sas "pciex1000,c1"
+mpt_sas "pciex1000,c2"
+mpt_sas "pciex1000,c3"
 mpt_sas "pciex1000,c4"
+mpt_sas "pciex1000,c5"
+mpt_sas "pciex1000,c6"
+mpt_sas "pciex1000,c7"
+mpt_sas "pciex1000,c8"
 mpt_sas "pciex1000,c9"
+mpt_sas "pciex1000,d0"
+mpt_sas "pciex1000,d1"
+mpt_sas "pciex1000,d2"
 mr_sas "pciex1000,52"
 mr_sas "pciex1000,53"
 mr_sas "pciex1000,5b"
